app/web/features/donations/DonationsBox.tsx

```diff
@@ -220,12 +220,21 @@ export default function DonationsBox() {
 
   const customAmountInput = useRef<HTMLInputElement>(null);
 
+  const checkForValidAmount: boolean = (amount: number) => {
```

Comment:
```suggestion
  const checkForValidAmount = (amount: number) => {
```
This type error is failing the build

Reply:
Thanks for your patience and the advice! I'll keep that in mind for the next issue. I've fixed the code and tried it, hopefuly it's all good now!